pathology/multiple_instance_learning/panda_mil_train_evaluate_pytorch_gpu.py

```diff
@@ -237,6 +237,7 @@ def list_data_collate(batch: collections.abc.Sequence):
     for i, item in enumerate(batch):
         data = item[0]
         data["image"] = torch.stack([ix["image"] for ix in item], dim=0)
+        data["patch_location"] = torch.stack([ix["patch_location"] for ix in item], dim=0)
         batch[i] = data
```

Comment:
l'm not sure should we also concatenate patch_size and offset to make the process more consistent?

Reply:
That's a good point, @KumoLiu, but patch_size and offset are identical for all the patches in a batch, so we need to only keep one of them instead creating an array of them like patch location.